### PR TITLE
Drop NAMateVFSURI

### PR DESCRIPTION
**Pull request by rbuj**

### CODE REVIEW

File: src/core/na-selected-info.c [hunk 1]
```diff
@@ -824,20 +823,24 @@ new_from_uri( const gchar *uri, const gchar *mimetype, gchar **errmsg )
...
-	na_mate_vfs_uri_parse( vfs, uri );
-	if( !info->private->filename ){
-		g_debug( "na_selected_info_new_from_uri: uri='%s', filename=NULL, setting it to '%s'", uri, vfs->path );
-		info->private->filename = g_strdup( vfs->path );
+	g_uri_split (uri,
+	             G_URI_FLAGS_NONE,
+	             &info->private->scheme,
+	             &info->private->username,
+	             &info->private->hostname,
+	             info->private->port,
```
Comment (raveit65) [inline src/core/na-selected-info.c]: build warning. https://app.travis-ci.com/github/mate-desktop/caja-actions/jobs/553895961#L4783
```
na-selected-info.c:831:15: warning: incompatible integer to pointer conversion passing 'gint' (aka 'int') to parameter of type 'gint *' (aka 'int *'); take the address with & [-Wint-conversion]

                     info->private->port,

                     ^~~~~~~~~~~~~~~~~~~

                     &

/usr/include/glib-2.0/glib/guri.h:99:52: note: passing argument to parameter 'port' here

                                     gint         *port,

                                                   ^
```

Comment (rbuj) [reply src/core/na-selected-info.c]: done

**Review by raveit65 — APPROVED** [review]
LGTM

**Review by mbkma — APPROVED** [review]
Builds & runs fine.

**Pull request merged**